flake.nix

```diff
@@ -95,6 +95,9 @@
               # rust toolchain
               (toolchain pkgs)
 
+              # for `hostname -I`
+              hostname-debian
```

Comment:
Is this added just for convenience, or is there a need for this in the development shell?

Reply:
required here: https://github.com/movementlabsxyz/movement/blob/main/scripts/movement/run#L23